### PR TITLE
[Merged by Bors] - chore(Order/SuccPred/CompleteLinearOrder): spacing fixes

### PR DESCRIPTION
**Pull request by vihdzp**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/SuccPred/CompleteLinearOrder.lean [hunk 1]
```diff
@@ -122,24 +117,23 @@ alias csSup_mem_of_not_isSuccLimit' := csSup_mem_of_not_isSuccPrelimit'
...
   · exact hs.mem_of_nonempty_of_not_isSuccPrelimit hs' hx
 
 @[deprecated IsLUB.mem_of_not_isSuccPrelimit (since := "2024-09-05")]
 alias IsLUB.mem_of_not_isSuccLimit := IsLUB.mem_of_not_isSuccPrelimit
 
-lemma IsLUB.exists_of_not_isSuccPrelimit (hf : IsLUB (Set.range f) x) (hx : ¬ IsSuccPrelimit x) :
-    ∃ i, f i = x := hf.mem_of_not_isSuccPrelimit hx
+lemma IsLUB.exists_of_not_isSuccPrelimit (hf : IsLUB (range f) x) (hx : ¬ IsSuccPrelimit x) :
+    ∃ i, f i = x :=
+  hf.mem_of_not_isSuccPrelimit hx
```
Comment (Command-Master) [inline Mathlib/Order/SuccPred/CompleteLinearOrder.lean:136]: According to the style guide using a single line is OK for a short proof

Comment (grunweg) [reply Mathlib/Order/SuccPred/CompleteLinearOrder.lean:136]: I also think the current way is fine.

**Review by grunweg — COMMENTED** [review]
All of these look fine to me. Perhaps not high-value to *find*, but I'd take them --- and as you're probably the person working with these files the most, you'll know how to deal with the conflicts. Thanks!
maintainer merge

**Comment by kim-em** [comment]
bors merge

**Pull request closed without merging**